culture/remote/README.md

```diff
@@ -0,0 +1,43 @@
+# Working with our Remote Teammates
+
+At Sparkbox, we have the opportunity to work with many different people in many different locations. Whether its' with our co-workers or our clients, sometimes, we all have to work remote.
+
+## Tips for Non-remote folks When Working With Remote Colleagues
+
+Let's be more inclusive of our remote coworkers and clients. Here are some ways how.
```

Comment:
Using `co-workers` on line 3 and `coworkers` here, I'm not sure which is correct but we can choose one